ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/transaction/jts/jca/TransactionImporterImple.java

```diff
@@ -171,7 +172,13 @@ public void removeImportedTransaction (Xid xid) throws XAException
 	 * @param timeout
 	 * @return
 	 */
-	private TransactionImportResult addImportedTransaction(TransactionImple recoveredTransaction, Xid mapKey, Xid xid, int timeout) {
+	private TransactionImportResult addImportedTransaction(TransactionImple recoveredTransaction, Xid mapKey, Xid xid, int timeout) throws XAException {
```

Comment:
Is it possible to consider removing the changes to this from this pull request as it is expected to be implemented in https://issues.redhat.com/browse/JBTM-3893 instead?

Reply:
Yep, sorry for the noise (probably, the IDE again :facepalm: )

Reply:
Sorry, I didn't realise there was that throws `XAException`. Removed